Makefile

```diff
@@ -460,6 +462,20 @@ build-archive:
 release-unix: clean full build-archive
 	@if [ -f e/Makefile ]; then $(MAKE) -C e release; fi
 
+.PHONY: release-darwin-unsigned
+release-darwin-unsigned: RELEASE:=$(RELEASE)-unsigned
+release-darwin-unsigned: clean full build-archive
+
+.PHONY: release-darwin
+release-darwin: release-darwin-unsigned
+	mkdir -p $(BUILDDIR)/
+	cp ./build.assets/build-common.sh ./build.assets/notarize-binaries.sh $(BUILDDIR)/
```

Comment:
Why do you have to copy these scripts? Can you just run them where they already are?